Formula/redpanda.rb

```diff
@@ -10,19 +10,21 @@ class Redpanda < Formula
   on_macos do
     if Hardware::CPU.arm?
       url "https://github.com/redpanda-data/redpanda/releases/download/v24.1.6/rpk-darwin-arm64.zip"
-      sha256 "5b9252ebe44469c399e37223bfdefc8ef69b8e0db555af2c802aef73e04dd441"
+      sha256 "4c7b01f508731c79ff35f8a9c7aeb6645d1b548c4a1e419694415f809c33c1e8"
```

Comment:
i'm getting a different shasum:
```console
bash$ curl -s -L https://github.com/redpanda-data/redpanda/releases/download/v24.1.6/rpk-darwin-arm64.zip | shasum -a 256 -
0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5  -
```

Reply:
Thank you so much for diligently checking this.  Could you check the file size?   This is a draft release, so auth should be needed for downloading it.

Reply:
ah, yes, the draft release asset has the matching shasum:
```console
bash$ shasum -a 256 *.zip
4c7b01f508731c79ff35f8a9c7aeb6645d1b548c4a1e419694415f809c33c1e8  rpk-darwin-arm64.zip
```